bootstrap/postgres/configs.go

```diff
@@ -451,12 +454,33 @@ func (cr configRepository) RemoveChannel(ctx context.Context, id string) error {
 	return nil
 }
 
+func (cr configRepository) ConnectThing(ctx context.Context, channelID, thingID string) error {
+	q := `UPDATE configs SET state = $1 WHERE EXISTS (
```

Comment:
No need for subquery have a look at https://github.com/absmach/magistrala/blob/31d0e8c4529339a6626c9ff03a65972bf407dd89/pkg/clients/postgres/clients.go#L27C1-L54C2

Reply:
I think we need the subquery as the ```configs``` table does not have a ```channel_id``` column.